changelogs/current.yaml

```diff
@@ -68,6 +68,17 @@ minor_behavior_changes:
   change: |
     Add more conversion in the proxy status utility. It can be disabled by the runtime guard
     ``envoy.reloadable_features.proxy_status_mapping_more_core_response_flags``.
+- area: http3
+  change: |
+    Use GRO (Generic Receive Offload) for reading packets from a client QUIC UDP socket. See
+    https://www.kernel.org/doc/html/next/networking/segmentation-offloads.html for a description of
+    GRO. This behavior change can be reverted by setting
+    ``envoy.reloadable_features.prefer_quic_client_udp_gro`` to ``false``.
+- area: http3
+  change: |
+    Allow recvmmsg (multi-message) to be used for reading packets from a client QUIC UDP socket, if
```

Comment:
wait I thought we wanted to disallow client recvmmsg?

Reply:
i changed the runtime guard to `disallow_` so the default behavior is to disallow recvmmsg.  PTAL